src/lib.rs

```diff
@@ -35,6 +35,18 @@ unsafe impl<T> Filled for T where T: Bitsized + From<<T as Bitsized>::ArbitraryI
 /// This is generated to statically validate that a type implements `FromBits`.
 pub fn assume_filled<T: Filled>() {}
 
+#[non_exhaustive]
+#[derive(Debug, PartialEq)]
+pub struct BitsError;
+
+/// Internally used for generating the `Result::Err` type in `TryFrom`.
+/// 
+/// This is needed since we don't want users to be able to create `BitsError` right now.
+/// We'll be able to turn `BitsError` into an enum later, or anything else really.
+pub fn give_me_error() -> BitsError {
+    BitsError
+}
```

Comment:
I would like a less whimsical function name here, since this appears in user's code (so no fun allowed)

Reply:
I think the dumb name is good since it discourages use and I like fun